src/schedule-and-details/license-section/license-selector/index.jsx

```diff
@@ -66,7 +66,7 @@ LicenseSelector.defaultProps = {
 LicenseSelector.propTypes = {
   licenseType: PropTypes.oneOf(Object.values(LICENSE_TYPE)),
   onChangeLicenseType: PropTypes.func.isRequired,
-  isEditable: PropTypes.bool,
+  isEditable: PropTypes.bool.isRequired,
```

Comment:
You can remove `isEditable` from the `defaultProps` section if it is now required.